### ISSUE TITLE
CameraRollModule.java uses or overrides a deprecated API.

### ISSUE THREAD

**Issue by palkerecsenyi** [react-native-community/react-native-cameraroll#93]
# Bug

I recently upgraded React Native to 0.61.2 from 0.60.4, and when I tried to run `react-native run-android` (after having run `gradlew clean`) I got the following error:

```
Note: C:\Users\Pal Kerecsenyi\Documents\Projects\PalChat\node_modules\@react-native-community\cameraroll\android\src\main\java\com\reactnativecommunity\cameraroll\CameraRollModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
```

## Environment info

React native info output:

```bash
 System:
    OS: Windows 10
    CPU: (4) x64 Intel(R) Core(TM) i5-4690K CPU @ 3.50GHz
    Memory: 914.50 MB / 7.94 GB
  Binaries:
    Node: 12.4.0 - C:\Program Files\nodejs\node.EXE
    npm: 6.9.0 - C:\Program Files\nodejs\npm.CMD
  SDKs:
    Android SDK:
      API Levels: 28, 29
      Build Tools: 28.0.3, 29.0.2
      System Images: android-29 | Google APIs Intel x86 Atom
  npmPackages:
    react: 16.9.0 => 16.9.0
    react-native: 0.61.2 => 0.61.2
```

Library version: 1.2.1
## Steps To Reproduce

1. Upgrade React Native 0.60.4 -> 0.61.2 (following 
2. Run `react-native run-android`

## Reproducible sample code

`package.json`

```json
{
  "name": "PalChat",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "start": "node node_modules/react-native/local-cli/cli.js start",
    "test": "jest"
  },
  "dependencies": {
    "@react-native-community/async-storage": "^1.6.2",
    "@react-native-community/cameraroll": "^1.2.1",
    "@react-native-community/geolocation": "^1.4.2",
    "@react-native-community/google-signin": "^3.0.0",
    "@types/react-native": "^0.60.17",
    "algoliasearch": "^3.33.0",
    "fast-sort": "^1.5.6",
    "geofirestore": "^3.3.1",
    "jetifier": "^1.6.1",
    "jsc-android": "^241213.1.0",
    "libphonenumber-js": "^1.7.22",
    "lodash": "^4.17.15",
    "moment": "^2.24.0",
    "react": "16.9.0",
    "react-devtools": "^3.6.3",
    "react-instantsearch": "^5.7.0",
    "react-instantsearch-dom": "^5.7.0",
    "react-instantsearch-native": "^5.7.0",
    "react-native": "0.61.2",
    "react-native-animate-number": "^0.1.2",
    "react-native-autogrow-textinput": "^5.2.0",
    "react-native-console-time-polyfill": "^1.2.1",
    "react-native-contacts": "^4.0.2",
    "react-native-country-picker-modal": "^0.8.0",
    "react-native-device-info": "^2.3.2",
    "react-native-exception-handler": "^2.10.8",
    "react-native-firebase": "^5.5.5",
    "react-native-flags": "^1.0.0",
    "react-native-fs": "^2.14.1",
    "react-native-geocoder": "^0.5.0",
    "react-native-gesture-handler": "^1.3.0",
    "react-native-image-picker": "^1.0.2",
    "react-native-image-resizer": "^1.0.1",
    "react-native-image-zoom-viewer": "^2.2.26",
    "react-native-inappbrowser-reborn": "^2.0.4",
    "react-native-launch-navigator": "^1.0.6",
    "react-native-linear-gradient": "^2.5.6",
    "react-native-modal-datetime-picker": "^7.5.0",
    "react-native-navigation-bar-color": "^0.1.0",
    "react-native-paper": "^3.0.0-alpha.3",
    "react-native-quick-actions": "^0.3.12",
    "react-native-reanimated": "^1.1.0",
    "react-native-select-multiple": "git+https://github.com/MnaveedS/react-native-select-multiple.git",
    "react-native-shimmer": "^0.5.0",
    "react-native-stars": "^1.1.5",
    "react-native-swiper": "^1.5.14",
    "react-native-text-input-mask": "^1.0.4",
    "react-native-vector-icons": "^6.5.0",
    "react-navigation": "^3.11.0",
    "react-navigation-fluid-transitions": "^0.3.2",
    "react-navigation-material-bottom-tabs": "^1.0.0",
    "react-navigation-transitions": "^1.0.12"
  },
  "devDependencies": {
    "@babel/core": "^7.5.5",
    "@babel/runtime": "^7.5.5",
    "@react-native-community/eslint-config": "^0.0.5",
    "babel-jest": "^24.8.0",
    "eslint": "^6.5.1",
    "hermes-engine": "^0.1.0",
    "jest": "^24.8.0",
    "metro-react-native-babel-preset": "^0.55.0",
    "react-test-renderer": "16.9.0"
  },
  "jest": {
    "preset": "react-native"
  }
}
```

**Comment by Gjoshi3107** [comment]
Actually i also faced this issue
It's occuring as [Latitude](https://developer.android.com/reference/android/provider/MediaStore.Images.ImageColumns#LATITUDE), [Longitude](https://developer.android.com/reference/android/provider/MediaStore.Images.ImageColumns#LONGITUDE) and [MediaStore.MediaColumns.DATA](https://developer.android.com/reference/android/provider/MediaStore.MediaColumns.html#DATA) is deprecated in API level 29.

**Comment by KevinGbalay** [comment]
> Actually i also faced this issue
> It's occuring as [Latitude](https://developer.android.com/reference/android/provider/MediaStore.Images.ImageColumns#LATITUDE), [Longitude](https://developer.android.com/reference/android/provider/MediaStore.Images.ImageColumns#LONGITUDE) and [MediaStore.MediaColumns.DATA](https://developer.android.com/reference/android/provider/MediaStore.MediaColumns.html#DATA) is deprecated in API level 29.

As Gjoshi3107 said, API 29 removed the ability to get location from images. 
A fix would be to remove them from the `CameraRollModule.java` file if location information is irrelevant to you. 
Remove `Images.Media.LONGITUDE` and `Images.Media.LATITUDE` from the java file, then either set `int longitudeIndex = 0;` and `int latitudeIndex = 0;` or remove them altogether. 
If you chose to remove them, make sure to remove the `putLocationInfo()` function and its call and you should be good.

**Comment by tulga1970** [comment]
Would this bug be fixed after this PR is merged https://github.com/react-native-community/react-native-cameraroll/pull/114 ?

**Comment by lordenas** [comment]
I have the same error. Is there a solution for this problem?

**Comment by rafaelcavalcante** [comment]
> I have the same error. Is there a solution for this problem?

I've forked this repo and supressed the latlong information which makes the application crash. You can install via npm/yarn using this repo: https://github.com/rafaelcavalcante/react-native-cameraroll

**Comment by syedrahaman92** [comment]
> Would this bug be fixed after this PR is merged #114 ?

Yes, This bug will be fixed after #114 PRis merged

**Comment by bartolkaruza** [comment]
Closing as fixed, thanks for reporting
